app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -60,6 +60,9 @@ class Config {
 
   DatastoreDB get db => _db;
 
+  // TODO(keyonghan): [FlutterBranches] should be refreshed periodically, https://github.com/flutter/flutter/issues/54315
+  Future<String> get flutterBranches => _getSingleValue('FlutterBranches');
+
```

Comment:
Seems like branching management is getting more and more complicated have you considered adding a branching service + provider class?

Reply:
Actually it is not. 
Multiple APIs (around 5) are reading Github branch API via cronjobs every one to five minutes. That would be more than one thousand reads per day.
Now, with this info. saved in datastore, we only need to refresh that property via a cronjob. This cronjob can be called every hour, or even less frequently, since release branches are generated every couple of days.